### DISCUSSION TITLE
How can I get Starstruck achievement in github?

### DISCUSSION THREAD

**Discussion by KingDev1026** [community/community#38664] [Other Feature Feedback, Questions, & Ideas]
don't know how I can get Quickdraw?

**Comment by hotcoding85** [comment]
please make this one:
https://github.com/hotcoding85/ckeditor5
btw i've done with yours....

**Comment by anirudhmsu** [comment]
Close an issue on a repo that you have access to, within 5 minutes.

**Comment by Sandeshkadel** [comment]
**1.Step-by-Step Process to Unlock “Starstruck”**
Step 1: Log into GitHub

Go to https://github.com/

Make sure you’re logged into the account where you want the achievement.

Step 2: Find a Repository to Star

You can star any public repository on GitHub.

Examples:

Popular open-source repos:

tensorflow/tensorflow

reactjs/react

torvalds/linux

Or repositories you follow, use, or contribute to.

Tip: You can search using the top search bar: type keywords like Python or JavaScript, then filter by repositories.

Step 3: Star the Repository

On the repository page, locate the “Star” button near the top-right corner (next to “Watch” and “Fork” buttons).

Click “Star”.

You should see the button change to “Unstar,” confirming you’ve starred it.

GitHub counts this action as your first star.

Step 4: Wait for the Achievement Badge to Appear

GitHub achievements update automatically, but sometimes it may take a few minutes.

To check:

Go to your profile page: [https://github.com/your-username](https://github.com/)

Scroll down to Achievements (usually near Contributions or Repositories).

You should see “Starstruck” unlocked.

**2. Extra Notes**

You only need to star one repository for Starstruck.

Subsequent stars do not give additional “Starstruck” badges.

Other achievements are related to different actions:

Arctic Code Vault Contributor – contribute to archived repos

Pair Extraordinaire – pair on a pull request

Pull Shark – open a pull request

If you’re not seeing it:

Make sure your account is verified and public. Private accounts sometimes delay achievements.

Refresh your profile or log out and back in.

Make sure you actually clicked Star (not just visited the page).

**Comment by kartik12421** [comment]
Go to any repositories and click the ⭐ “Star” button — repeat until you hit milestones (like 16 stars) and GitHub will automatically give you the Starstruck achievement.

**Reply by melaxman** [reply]
ok

**Comment by Sumanth0019** [comment]
The best and fastest approach is to go to https://www.fenrir.codes/ and add whatever repository you have. You can then give stars to other repositories and receive stars for your own repository in a few days.  You can obtain the achievement if you have about 16 stars.

**Reply by melaxman** [reply]
oo

**Comment by Fatiihunza89** [comment]
The Starstruck achievement on GitHub is earned when one of your public repositories receives stars from other GitHub users.

How to get it:
Create and maintain a useful public repository.
Share your project with the community.
Write a clear README and documentation.
Promote your repository on GitHub, social media, or developer communities.
Build something that others find valuable and want to star.

The more stars your repository receives, the higher the Starstruck achievement level you can unlock.

Just as GitHub rewards projects that provide value to the community, websites like [Bedsheet.com.pk](https://bedsheet.com.pk/?utm_source=chatgpt.com) grow by offering quality products and a great user experience.

**Comment by lgustavopuga** [comment]
To unlock these achievements, follow these steps:

### 1. Starstruck Achievement

You earn this badge when a repository you own reaches a specific number of stars from other GitHub users:

* **Bronze (First tier):** 16 stars
* **Silver:** 128 stars
* **Gold:** 512 stars

**Tips to get it:**

* Create a useful project or template.
* Write a clear, engaging README with screenshots or setup steps.
* Share your repository in developer communities, social media, or Discord.

### 2. Quickdraw Achievement

You mentioned Quickdraw in your comment. You earn this badge by closing an issue or a pull request (PR) extremely fast.

* **How to get it:** Close an issue or a PR within **5 minutes** of its creation.
* **Quickest way:** Go to one of your own repositories, create a test issue, and immediately click "Close issue."

Both badges will automatically appear on your GitHub profile once you meet these requirements.

**Comment by Hilal-11** [comment]
> don't know how I can get Quickdraw?

Starstruck is earned when a repository you own gets 16+ stars.

Tiers: Bronze (16 stars) → Silver (128) → Gold (1,024) — more stars unlock higher tiers.
It only counts stars on repos you personally own (not orgs, not forks you don't own).
No trick to fake it — you need real people starring your repo. Sharing genuinely useful projects (tools, templates, guides) on Twitter/Reddit/dev communities is the realistic way to get there.
